OWNERS

```diff
@@ -5,9 +5,11 @@ filters:
     - slintes
     - n1r1
```

Comment:
Nit: I suggest also removing Nir (n1r1) and adding Or.

Reply:
Instead of removing Nir maybe we add him as `emeritus_approvers` like in NHC https://github.com/medik8s/node-healthcheck-operator/blob/main/OWNERS_ALIASES#L18-L20? @mshitrit